specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2697,6 +3047,167 @@
       ],
       "description": "Parameter required for creating a linked server to redis cache."
     },
+    "RedisCacheAccessPolicyProperties": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of access policy",
+          "readOnly": true,
+          "enum": [
+            "Updating",
+            "Succeeded",
+            "Deleting",
+            "Deleted",
+            "Canceled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "AccessPolicyProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "type": {
+          "type": "string",
+          "description": "Built-In or Custom access policy",
```

Comment:
>Custom [](http://example.com/codeflow?start=38&length=6)

nit, lowercase 'custom'

Reply:
Kept Custom with a capital C here due to the enum values for type being "Built-In" and "Custom"

Reply:
I think maybe that's a little irrelevant, since just listing the available values of an enum isn't the main job of the property description - the point is to describe the meaning of the 'type' property. But if you want to go that way, you could clarify that you're quoting, and not just writing ordinary words, by e.g. 

```The kind of access policy, which can be either 'Built-In' (predefined) or 'Custom'```